backend/service/k8s/configmap.go

```diff
@@ -0,0 +1,47 @@
+package k8s
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+
+	k8sapiv1 "github.com/lyft/clutch/backend/api/k8s/v1"
+)
+
+func (s *svc) ListConfigMaps(ctx context.Context, clientset, cluster, namespace string, listOptions *k8sapiv1.ListOptions) ([]*k8sapiv1.ConfigMap, error) {
+	cs, err := s.manager.GetK8sClientset(clientset, cluster, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	opts := ApplyListOptions(listOptions)
```

Comment:
This is not directly related to this PR, but currently `ApplyListOptions` does not support fieldselector. If we expand this method now we will be able to use this to get specific configmaps based on name, which is currently not possible. We should probably do a follow up PR to enable `ApplyListOptions` to support fieldselector

Reply:
Agree, will create a ticket to follow-up on!